apps/browser-extension-wallet/src/components/MainMenu/DropdownMenuOverlay/components/RenameWalletDrawer.tsx

```diff
@@ -0,0 +1,168 @@
+          accountIndex: account.value.accountIndex,
+          metadata: { ...account.value.metadata, name: account.value.metadata.name }
+        });
+      }
+
+      void posthog.sendEvent(PostHogAction.RenameWalletSaveClick);
+      toast.notify({
+        duration: TOAST_DEFAULT_DURATION,
+        text: t('browserView.renameWalletDrawer.renameSuccessful')
+      });
+    } catch (error) {
+      logger.error(error);
+      toast.notify({
+        duration: TOAST_DEFAULT_DURATION,
+        text: t('browserView.renameWalletDrawer.renameFailed')
+      });
+    } finally {
+      onClose();
+    }
+  };
+
+  const handleOnChangeAccountData = (event: Readonly<React.ChangeEvent<HTMLInputElement>>, index: number) => {
+    setAccountsData((prev) => {
+      const currentAccountData = [...prev];
+      currentAccountData[index].value.metadata.name = event.target.value;
+      currentAccountData[index].hasError = !isValidInputLength(event.target.value);
+      return currentAccountData;
+    });
+  };
+
+  const handleOnClose = () => {
+    void posthog.sendEvent(PostHogAction.RenameWalletCancelClick);
+    onClose();
+  };
+
+  return (
+    <Drawer
+      open={open}
+      onClose={handleOnClose}
+      title={<DrawerHeader popupView={popupView} title={t('browserView.renameWalletDrawer.title')} />}
+      navigation={
+        <DrawerNavigation
+          title={t('browserView.renameWalletDrawer.walletSettings')}
+          onCloseIconClick={!popupView ? handleOnClose : undefined}
+          onArrowIconClick={popupView ? handleOnClose : undefined}
+        />
+      }
+      footer={
+        <Flex flexDirection="column" gap="$16">
+          <Button.CallToAction
+            label={t('browserView.renameWalletDrawer.save')}
+            w="$fill"
+            onClick={renameWallet}
+            disabled={isSaveButtonDisabled}
+          />
+          <Button.Secondary label={t('browserView.renameWalletDrawer.cancel')} w="$fill" onClick={handleOnClose} />
+        </Flex>
+      }
+      popupView={popupView}
+    >
+      <>
+        <Box mt="$44">
+          <Box mb="$24">
+            <Text.Body.Large color="secondary">{t('browserView.renameWalletDrawer.renameWallet')}</Text.Body.Large>
+          </Box>
+          <TextBox
+            label={t('browserView.renameWalletDrawer.walletName')}
+            value={newWalletName.value}
+            onChange={(e) => setNewWalletName({ value: e.target.value, hasError: !isValidInputLength(e.target.value) })}
+            w="$fill"
+            errorMessage={newWalletName.hasError ? errorMessage : undefined}
+          />
+        </Box>
+
+        {isAnyBip32Wallet(wallet) && (
+          <Box mt="$60" mb="$16">
+            <Box mb="$24">
+              <Text.Body.Large color="secondary">
+                {t('browserView.renameWalletDrawer.renameEnabledAccounts')}
+              </Text.Body.Large>
+            </Box>
+            <Box className={styles.inputGroupContainer}>
```

Comment:
```suggestion
           <Flex flexDirection="column" gap="$16">
```
please remove `apps/browser-extension-wallet/src/components/MainMenu/DropdownMenuOverlay/components/RenameWalletDrawer.module.scss` if that works